app/assets/stylesheets/variables/_app.scss

```diff
@@ -1,7 +1,6 @@
 $serif-font-family: 'Merriweather Web', 'Georgia', 'Cambria', 'Times New Roman', 'Times', serif !default;
 $sans-serif-font-family: 'Source Sans Pro Web', 'Helvetica Neue', 'Helvetica', 'Roboto', 'Arial',
   sans-serif !default;
-$monospace-font-family: 'Source Code Pro', 'Consolas', monospace !default;
```

Comment:
Fun fact: We specify "Source Code Pro" as the preferred font, but we don't actually make it available for download, so it's currently using whichever system font the device has available (Courier on macOS).